frontend/src/lib/Course.svelte

```diff
@@ -32,20 +33,32 @@
 </script>
 
 <div>
-  <h3 class="text-sm font-bold">
-    <span title={data.subjectDescription}>{data.subject}</span>
-    {data.catalogNumber}:
-    {data.title || "[No Title]"} ({data.semester})
-  </h3>
-  <p class="text-sm mb-1">
-    {#each data.instructors as instructor, i}
-      <a
-        href={instructor.email
-          ? `mailto:${instructor.email.toLowerCase()}`
-          : undefined}>{instructor.name}</a
-      >{#if i < data.instructors.length - 1}{", "}{/if}
-    {/each}
-  </p>
+  <div class="flex gap-2">
+    <div class="flex-1">
+      <h3 class="text-sm font-bold">
+        <span title={data.subjectDescription}>{data.subject}</span>
+        {data.catalogNumber}:
+        {data.title || "[No Title]"} ({data.semester})
+      </h3>
+      <p class="text-sm mb-1">
+        {#each data.instructors as instructor, i}
+          <a
+            href={instructor.email
+              ? `mailto:${instructor.email.toLowerCase()}`
+              : undefined}>{instructor.name}</a
+          >{#if i < data.instructors.length - 1}{", "}{/if}
+        {/each}
+      </p>
+    </div>
+    <div class="mt-0.5 hidden sm:block">
+      <GenEdAreas value={data.genEdArea} />
+    </div>
+  </div>
+  {#if data.genEdArea.length}
+    <div class="mb-2 sm:hidden">
```

Comment:
This makes the bottom margin slightly larger than the top. Here's `mb-1`, which I think packs the course listing together slightly better:
![screenshot-mb-1](https://github.com/ekzhang/classes.wtf/assets/16979554/9289c3bd-0dd2-4cc3-9b78-952a109c0a73)

Reply:
I think it still needs the space, maybe we can settle on 6px